src/Modules/UngradedModules/SubquoModuleElem.jl

```diff
@@ -1000,7 +1000,8 @@ function is_zero(m::SubquoModuleElem)
   return (ambient_representative(m) in parent(m).quo)
 end
 
-function iszero(m::SubquoModuleElem{<:MPolyRingElem})
+# Method only for the cases which can be handled by singular
+function iszero(m::SubquoModuleElem{<:MPolyRingElem{T}}) where {T<:Union{<:FieldElem, ZZRingElem}}
```

Comment:
@jankoboehm : This function is not generic, but works only for rings with a singular backend. When writing up your different module layers, this should be corrected, eventually.